src/ghci/stdin.rs

```diff
@@ -122,41 +144,78 @@ impl GhciStdin {
     }
 
     #[instrument(skip_all, level = "debug")]
-    async fn initialize(&mut self, sender: oneshot::Sender<()>) -> miette::Result<()> {
-        // For Stack, send a blank line to skip an initial prompt.
-        //
-        // See: https://github.com/ndmitchell/ghcid/issues/57
-        self.stdin.write_all(b"\n").await.into_diagnostic()?;
```

Comment:
This caused a deadlock/race condition... sometimes the stdout task would've caught up with the prompt in time, sometimes it got stuck waiting for a prompt that never came. We can implement this again later (and I (filed a ticket for it)[https://linear.app/mercury/issue/DUX-1292/multi-project-stack-support]).